client-spec.md

```diff
@@ -263,7 +263,9 @@ While other signing and verification workflows are possible using the Transparen
 
 This section describes the “Sigstore wire format” for verification materials.
 
-To produce verification materials in this format, a client MUST use the Protocol Buffers [Bundle format](https://github.com/sigstore/protobuf-specs/blob/88c45b0ab8c3781a118be6339f443d8c277c0126/protos/sigstore_bundle.proto#L61-L77) to collate these materials, serialized to JSON using the [canonical proto3 JSON serialization](https://protobuf.dev/programming-guides/proto3/#json), *except* that:
+To produce verification materials in this format, a client MUST use the Protocol Buffers [Bundle format](https://github.com/sigstore/protobuf-specs/blob/88c45b0ab8c3781a118be6339f443d8c277c0126/protos/sigstore_bundle.proto#L61-L77) to collate these materials to ensure Sigstore tooling and implementations are compatible with each other. Verification materials MAY be also made available in other formats if needed by a specific use-case.
```

Comment:
Should we be more explicit that 3rd party tooling may transofmr bundles to and from other formats? Or do you think this is sufficient?

Reply:
I think it's sufficient as is 👍